extensions/wikia/FeedsAndPosts/Discussion/LinkHelper.php

```diff
@@ -68,6 +68,15 @@ public function buildThreadsLink( Uri $uri, IContextSource $requestContext ): st
 		return $this->buildLink( $uri, $controllerQueryParams );
 	}
 
+	public function buildPostsLink( Uri $uri, IContextSource $requestContext ): string {
```

Comment:
did you check if we don't return posts links in other responses? I suspect not, but just to be safe

Reply:
Ah, found one! Good catch, I need to fix it